doc/AddonStorageBackend.md

```diff
@@ -140,18 +140,18 @@ abstract class StorageTest
 
 There are two intended types of exceptions for storages
 
-### `ReferenceStorageExecption`
+### `ReferenceStorageException`
```

Comment:
If not breaking inbound links is of concern, you could duplicate the heading so that the misspelled variant will also point here as well at least during a transitional period of a year or so.

Reply:
Right. I can do that if requested.

Some quick checks:
* https://grep.app/search?q=ReferenceStorageExecption - 0 hits
* https://www.google.com/search?q=%22ReferenceStorageExecption%22&oq=%22ReferenceStorageExecption%22&aqs=chrome..69i57j0i546l3.622j0j4&sourceid=chrome&ie=UTF-8 - 1 hit - https://wiki.friendi.ca/docs/addonstoragebackend -- which is I think this document itself
* https://github.com/search?q=ReferenceStorageExecption&type=code - just this code, its forks, and this PR

So, offhand, this change should be safe w/o including anything.

Reply:
There shouldn't be any inbound external link to this header.